core/src/main/groovy/noe/tomcat/configure/ConnectorConfiguratorTomcat.groovy

```diff
@@ -124,29 +124,33 @@ class ConnectorConfiguratorTomcat {
   }
 
   private void updateExistingConnector(Node connector, Node newConnector) {
-    // update attributes
+    // update connector attributes
     newConnector.attributes().each { attribute ->
       connector.@"${attribute.key}" = attribute.value
     }
 
-    // update nodes
     newConnector.each { Node newSubelement ->
-      String UpgradeProtocol = ConnectorUpgradeProtocolTomcat.ELEMENT_NAME
```

Comment:
IIUC, we are losing the configuration of UpgradeProtocol, no?

Reply:
No UpgradeProtocol configuration remains the same and working properly

Reply:
This Element name was used in order to identify inner element for update.Now i "scan" new element's name and edit the specific one.Leaving the rest connector unchanged

Reply:
ok, please see my last comment below for details.